src/main/java/plugins/nate/smp/listeners/ChestLockListener.java

```diff
@@ -96,18 +110,31 @@ public void onHopperPlace(BlockPlaceEvent event) {
             }
         }
     }
-
+    /**
+    *   Spawn particles with random offsets, .25 blocks in variation
+    */
+    private void signCreationParticles(World world, Location location, Particle particle) {
```

Comment:
This method name isn't a verb, it's just `signCreationParticles` which seems more idiomatic of a variable name to me. This could benefit from a rename, maybe `showSignParticles` or something?

```suggestion
    private void showSignParticles(World world, Location location, Particle particle) {
```